.github/workflows/ci.yml

```diff
@@ -1,50 +1,61 @@
 name: CI
-'on':
+"on":
   push:
     branches:
       - master
     tags:
-      - 'v*'
+      - "v*"
   pull_request:
     branches:
       - master
 jobs:
+  Build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.13"
+      - uses: astral-sh/setup-uv@v3.2.0
+      - run: uv build .
+      - uses: actions/upload-artifact@v4
+        with:
+          name: dist
+          path: dist/
   Test:
-    runs-on: '${{ matrix.os }}'
+    runs-on: "${{ matrix.os }}"
     strategy:
       matrix:
         os:
           - ubuntu-22.04
         python-version:
-          - '3.7'
-          - '3.8'
-          - '3.9'
-          - '3.10'
+          - "3.7"
```

Comment:
We could maybe consider dropping 3.7 at some point?
(Although no need to do it now as things work.)